editor/import/resource_importer_layered_texture.cpp

```diff
@@ -357,6 +358,10 @@ Error ResourceImporterLayeredTexture::import(const String &p_source_file, const
 	if (channel_pack == 0) {
 		csource = Image::COMPRESS_SOURCE_SRGB;
 	}
+	else if( channel_pack == 2) {
+		// force normal
+		csource = Image::COMPRESS_SOURCE_NORMAL;
+	}
```

Comment:
Please follow the code style guidelines in place. You can do so locally by installing clang-format and the pre-commit hook: https://docs.godotengine.org/en/stable/community/contributing/code_style_guidelines.html#pre-commit-hook

Reply:
for now i manually i ran clang format with "https://github.com/godotengine/godot/blob/master/.clang-format" on the file.
this is ok right?